src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+class Ball {
+    private final Color color;
+    private final int number;
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball [color=" + color + ", number=" + number + "]";
+    }
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
```

Reply:
but if i remove the last blank line my git throws an error